src/SignalRServiceExtension/Bindings/SignalRInputBindings/SignalRConnectionInputBinding.cs

```diff
@@ -64,7 +67,21 @@ internal class SignalRConnectionInputBinding : BindingBase<SignalRConnectionInfo
             signalRConnectionInfoConfigurer.Configure(tokenResult, request, signalRConnectionDetail);
             var customizedInfo = azureSignalRClient.GetClientConnectionInfo(signalRConnectionDetail.UserId,
                 signalRConnectionDetail.Claims);
-            return Task.FromResult((IValueProvider)new SignalRValueProvider(customizedInfo));
+            return Task.FromResult<IValueProvider>(new SignalRValueProvider(ConvertToUserType(customizedInfo), userType, ""));
```

Comment:
>ConvertToUserType(customizedInfo) [](start = 76, length = 33)

if real type (userType) is passed to SignalRValueProvider, shall this convert done inside SignalRValueProvider?